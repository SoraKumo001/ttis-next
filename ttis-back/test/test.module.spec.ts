import * as fs from 'fs';
import * as path from 'path';

const execTest = (dir: string = __dirname) => {
  const files = fs.readdirSync(dir);
  files.forEach((file) => {
    const filePath = path.resolve(dir, file);
    const state = fs.statSync(filePath);
    if (state.isDirectory()) execTest(filePath);
    else /.*\.test\.ts$/.test(file) && require(filePath);
  });
};

execTest("src");
