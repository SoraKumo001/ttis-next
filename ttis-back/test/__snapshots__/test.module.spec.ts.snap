// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Contens(Login) Create & update 1`] = `
Object {
  "updateContents": Object {
    "__typename": "Contents",
    "createAt": "",
    "id": "",
    "page": true,
    "parentId": "",
    "priority": 1,
    "title": "UpdateTest",
    "title_type": 2,
    "updateAt": "",
    "value": "Test Value",
    "value_type": "TEXT",
    "visible": null,
  },
}
`;

exports[`Contens(Login) Create & update 2`] = `
Object {
  "updateContents": Object {
    "__typename": "Contents",
    "createAt": "",
    "id": "",
    "page": true,
    "parentId": "",
    "priority": 1,
    "title": "UpdateTest",
    "title_type": 2,
    "updateAt": "",
    "value": "Test Value",
    "value_type": "TEXT",
    "visible": null,
  },
}
`;

exports[`Contens(Login) Create contens 1`] = `
Object {
  "createContents": Object {
    "__typename": "Contents",
    "createAt": "",
    "id": "",
    "page": true,
    "parentId": "",
    "priority": 1,
    "title": "New",
    "title_type": 1,
    "updateAt": "",
    "value": "",
    "value_type": "HTML",
    "visible": null,
  },
}
`;

exports[`Contens(Login) Levels 1`] = `
Object {
  "data": Object {
    "updateContents": Object {
      "__typename": "Contents",
      "createAt": "",
      "id": "",
      "page": true,
      "parentId": "",
      "priority": 1,
      "title": "UpdateTest",
      "title_type": 2,
      "updateAt": "",
      "value": "Test Value",
      "value_type": "TEXT",
      "visible": null,
    },
  },
}
`;

exports[`Contens(Login) Query contens 1`] = `
Object {
  "contentsList": Object {
    "0": Object {
      "__typename": "Contents",
      "createAt": "",
      "id": "",
      "page": true,
      "parentId": "",
      "priority": 1000,
      "title": "TOP",
      "title_type": 1,
      "updateAt": "",
      "value": "",
      "value_type": "HTML",
      "visible": true,
    },
    "1": Object {
      "__typename": "Contents",
      "createAt": "",
      "id": "",
      "page": true,
      "parentId": "",
      "priority": 1,
      "title": "New",
      "title_type": 1,
      "updateAt": "",
      "value": "",
      "value_type": "HTML",
      "visible": null,
    },
    "2": Object {
      "__typename": "Contents",
      "createAt": "",
      "id": "",
      "page": true,
      "parentId": "",
      "priority": 1,
      "title": "UpdateTest",
      "title_type": 2,
      "updateAt": "",
      "value": "Test Value",
      "value_type": "TEXT",
      "visible": null,
    },
    "3": Object {
      "__typename": "Contents",
      "createAt": "",
      "id": "",
      "page": true,
      "parentId": "",
      "priority": 1,
      "title": "UpdateTest",
      "title_type": 2,
      "updateAt": "",
      "value": "Test Value",
      "value_type": "TEXT",
      "visible": null,
    },
  },
}
`;

exports[`Contens(Login) Query contens 2`] = `
Object {
  "contents": Object {
    "__typename": "Contents",
    "createAt": "",
    "id": "",
    "page": true,
    "parentId": "",
    "priority": 1,
    "title": "New",
    "title_type": 1,
    "updateAt": "",
    "value": "",
    "value_type": "HTML",
    "visible": null,
  },
}
`;

exports[`User(login) CreateUser 1`] = `
Object {
  "__typename": "User",
  "id": 2,
  "info": "{}",
  "name": "Test01",
}
`;

exports[`User(login) CurrentUser 1`] = `
Object {
  "__typename": "User",
  "info": "{}",
  "name": "admin",
}
`;

exports[`User(login) UpdateUser 1`] = `
Object {
  "data": Object {
    "updateUser": Object {
      "__typename": "User",
      "id": 2,
      "info": "{}",
      "name": "Test01-02",
    },
  },
}
`;

exports[`User(login) User List 1`] = `
Array [
  Object {
    "__typename": "User",
    "id": 1,
    "info": "{}",
    "name": "admin",
  },
]
`;
